masking-examples/consistent-format-preserve/oracle-complete-script.sql

```diff
@@ -7,6 +7,8 @@ GRANT EXECUTE ON DBMS_CRYPTO TO PUBLIC;
 CREATE OR REPLACE PACKAGE CYRAL.CYRALCUSTOMPKG IS
     FUNCTION "consistent_mask"(data IN VARCHAR2) RETURN VARCHAR2;
     FUNCTION "consistent_mask"(data IN NUMBER) RETURN NUMBER;
+    FUNCTION "consistent_mask_hash"(data IN VARCHAR2) RETURN VARCHAR2;
+    FUNCTION "consistent_mask_hash"(data IN NUMBER) RETURN NUMBER;
```

Comment:
Reminder to mention this implementation in the README.

Reply:
I see you've updated the README in the root dir, I meant the README in this directory.